src/ngraph/node.hpp

```diff
@@ -379,6 +379,8 @@ namespace ngraph
         /// Get all the nodes that uses the current node
         NodeVector get_users(bool check_is_used = false) const;
 
+        /// \return Version of this node
+        virtual size_t get_op_version() const { return 0; }
```

Comment:
`get_version` should be good enough since it's a method on the "op".

Reply:
Name was changed ;)